containers/EESSI-squid-proxy.def

```diff
@@ -0,0 +1,49 @@
+BootStrap: docker
+From: centos:7
```

Comment:
@bedroge Wouldn't it make more sense to make this a Dockerfile, so we can build the Docker image and push it to Docker Hub (and then we can also run it via Singularity)?

Reply:
Good idea! I'll convert it to a Dockerfile.

Reply:
One disadvantage is that you can't use `singularity instance` with a Docker image, since `startscript` is not defined in that case...